pyat/setup.py

```diff
@@ -31,6 +32,23 @@ def select_omp():
 
 cflags = []
 
+
+mpi = os.environ.get('MPI', None)
+if mpi is None:
+    mpi_macros = []
+    mpi_includes = []
```

Comment:
`mpi_macros` is used for compiling the integrators, but not for compiling `at.c`. As a consequence, `DConstant.mpi` always returns False !
Isn't there a macro automatically set by `mpicc` which could be used? it would be safer.

Reply:
Ah right! Well I looked very hard for it and could not find anything...I think we can compile `at.c` with `mpicc` as it is just a wrapper of `gcc`

Reply:
Corrected